spaceward/src/features/modals/ConnectedModal.tsx

```
@@ -5,11 +5,10 @@ import { SessionTypes } from "@walletconnect/types";
 import { useModalState } from "./state";
 import { ModalParams } from "./types";
 import AddressAvatar from "@/components/AddressAvatar";
+import { env } from "@/env";
 
 export default function ConnectedModal(props: ModalParams<{}>) {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)